Research/lva-node-app/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
After modifying this as we discussed earlier, did you regenerate the package-lock.json file? If not, could take care of that?

Reply:
just did!